python/feldera/pipeline.py

```diff
@@ -484,7 +484,9 @@ def stop(self, force: bool, timeout_s: Optional[float] = None):
             for view_name, queue in self.views_tx.pop().items():
                 # block until the callback runner has been stopped
                 queue.join()
+        import time
 
+        time.sleep(3)
```

Comment:
this is a fix?

Reply:
Needs a proper fix from @abhizer see commit comment